File: perl/tests/200-index-data-providers.pl
my $normal      = &NormalHandle;
our $dp_list_200 = $normal->index_data_providers();

if ($DEBUG) {
  my $top_entries = &ShowTopEntries($dp_list_200,4);
  print "LIST (first dp)=\n${top_entries}\n";
}

die "Value returned not an array.\n" unless defined($dp_list_200) && ref($dp_list_200) eq "ARRAY";

1;

File: ruby/tests/200-index-data-providers.rb
normal       = normal_handle
@dp_list_200 = normal.index_data_providers()

puts "LIST (first dp)= \n#{show_top_entries(@dp_list_200,5)}" if @debug

raise "Value returned is not an array" unless @dp_list_200.present? && @dp_list_200.is_a?(Array)

true
